inc/class_parser.php

```diff
@@ -1611,6 +1611,7 @@ function mycode_parse_video($video, $url)
 					// Channel (livestream) embed with URL like: https://twitch.tv/waypoint
 					$id = 'channel='.$path[1];
 				}
+				$parent_domain = parse_url($mybb->settings['bburl'], PHP_URL_HOST);
```

Comment:
It might be good to:
 - rename the variable to `$bbdomain`, and
 - initialize it as a _static_ variable somewhere in the beginning of the function.